src/main/resources/templates/ward.html

```diff
@@ -29,6 +29,54 @@ <h4>Ward</h4>
     <b>Code:</b> <span th:text="${ward.fullCode}"></span><br />
     <b>Name:</b> <span th:text="${ward.name}"></span><br />
   </div>
+
+  <h2>Polling Units</h2>
+  <div>
+    <table class="striped responsive-table">
+      <tr>
+        <th>#</th>
+        <th><a th:href="@{?sortBy=fullCode&sortDirection=ASC}">Code</a></th>
+        <th><a th:href="@{?sortBy=description&sortDirection=ASC}">Description</a></th>
+        <th><a th:href="@{?sortBy=longitude&sortDirection=ASC}">Longitude</a></th>
+        <th><a th:href="@{?sortBy=latitude&sortDirection=ASC}">Latitude</a></th>
+      </tr>
+      <tr th:each="pollingUnit, i: ${form.wardPage.content}">
+        <td th:text="${(form.wardPage.number*form.wardPage.size)+i.index+1}"></td>
+        <td class="whitespace">
```

Comment:
I still would advise against using styles classes like this

Reply:
What would you suggest?